app/views/application/_involved_person.html.erb

```diff
@@ -14,12 +14,14 @@
     <%= f.label :injury_type, person_type + " injury type - check all that apply", class: 'required' %>
     <%= checkbox_grid(f, :injury_type, InvolvedPerson::INJURY_TYPES, 180) %>
   </div>
-  <div class="incident-form-row form-group additional-question">
+  <div class="incident-form-row form-group additional-question medical-aid-received">
     <%= f.label :medical_aid, person_type + " medical aid received - choose highest applicable", class: 'required' %>
     <%= radio_button_grid(f, :medical_aid, InvolvedPerson::MEDICAL_AID, 350) %>
   </div>
-  <div class="incident-form-row form-group additional-question">
-    <%= yes_no_question f, :injury_from_preexisting_condition,  person_type + " injury caused by a pre-existing condition?", class: 'required' %>
+  <div class="incident-form-row form-group additional-question pre-existing-condition">
```

Comment:
For here and above, we had been using an id to identify very specific elements. Classes should be used to style or select groups of elements, but since we're trying to single out very specific ones, please use the id= attribute and id selectors in your js (e.g. `#medical-aid-received`)